docs/content/guides/author-apps/kubernetes/howto-yaml-base-support/snippets/basemanifest.bicep

```diff
@@ -0,0 +1,43 @@
+import radius as radius
+
+
+@description('Specifies the image of the container resource.')
+param magpieimage string = 'ghcr.io/radius-project/dev/magpiego:pr-498da3203f'
+
+@description('Specifies the port of the container resource.')
+param port int = 3000
+
+@description('Specifies the environment for resources.')
+param environment string
+
+@description('Specifies the application for resources.')
+param application string
+
+//CONTAINER
+@description('Loads the Kubernetes base manifest file and turns content into a string.')
+var manifest = loadTextContent('./basemanifest.yaml')
```

Comment:
Instead of having this a var let's just inline the loadTextContent down in the resource definition. Currently a user has to go back and forth to understand how this var is used and what it is, and it's only used in one location in the file.